lib/mrtrix3/run.py

```diff
@@ -330,9 +346,6 @@ def quote_nonpipe(item):
         line[0] = version_match(line[0])
         if shared.get_num_threads() is not None:
           line.extend( [ '-nthreads', str(shared.get_num_threads()) ] )
-        # Get MRtrix3 binaries to output additional INFO-level information if script running in debug mode
-        if shared.verbosity == 3 and not any(entry in line for entry in ['-info', '-debug']):
-          line.append('-info')
```

Comment:
All other changes in this PR are made in order to facilitate this deletion, which was the cause of the error reported in #2268.